src/material/sort/sort-header.scss

```diff
@@ -31,6 +31,7 @@ $mat-sort-header-arrow-hint-opacity: 0.38;
   outline: 0;
   font: inherit;
   color: currentColor;
+  position: relative;
```

Comment:
For the components you made `relative` in this change: could you make a CL (or three CLs if you want to be granular) with just that `relative` change and run through global presubmit to see if any screenshot diffs pop up? That's typically our litmus test for this type of style change.

Reply:
Yes, will do.

Reply:
Ran through global presubmit and all looked good.